src/composables/useSidebar.ts

```
@@ -62,7 +77,7 @@
   defaultTag: 'Default',
 } as const
 
-function defaultGroupTemplate(path: string, _: number = 1): string {
+function defaultGroupTemplate({ path, depth }: SidebarGroupTemplateParams): string {
```

ESLint

[error] 'depth' is defined but never used. Allowed unused args must match /^_/u.

(unused-imports/no-unused-vars)